applications/FluidDynamicsApplication/tests/test_mass_conservation_check_process.py

```diff
@@ -0,0 +1,229 @@
+﻿from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utils
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+from KratosMultiphysics.FluidDynamicsApplication.apply_mass_conservation_check_process import ApplyMassConservationCheckProcess
+import sys
+import re
+
+try:
+    import KratosMultiphysics.mpi as KratosMPI
+    import KratosMultiphysics.MetisApplication as KratosMetis           
+    import KratosMultiphysics.TrilinosApplication as KratosTrilinos
+    import KratosMultiphysics.mpi.distributed_import_model_part_utility as distributed_import_model_part_utility
+except:
+    print("Running tests without MPI")
+
+def GetFilePath(fileName):
+    return KratosMultiphysics.os.path.dirname(KratosMultiphysics.os.path.realpath(__file__)) + "/" + fileName
+
+class TestMassConservationCheckProcess(KratosUnittest.TestCase):
+    def setUp(self):
+        self.comm = KratosMultiphysics.DataCommunicator.GetDefault()
+        self.size = self.comm.Size()
+        self.rank = self.comm.Rank()
+
+    def tearDown(self):
+        with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
+            self.comm.Barrier()
+            if self.rank == 0:
+                kratos_utils.DeleteFileIfExisting(self.file_name + ".time")
+            kratos_utils.DeleteFileIfExisting(self.file_name + "_" + str(self.rank) + ".mdpa")
+            self.comm.Barrier()
+
+    def _SetParameters(self, name, use_memory):
+        parameters = """{
+            "echo_level" : 0,
+            "model_import_settings" : {
+                "input_type" : "mdpa",
+                "input_filename" :\""""+ name +"""\",
+                "partition_in_memory" : """ + use_memory + """
```

Comment:
this doesn't work in Win unfortunately

Reply:
it does, I actually program in windows most of the time.

Reply:
strange
I use it like this [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MappingApplication/tests/mapper_test_case.py#L147-L156) and this doesn't work in Win

do you know why?

Reply:
This also works for me in windows. In fact, we have been using this for months in @KratosMultiphysics/altair and it has worked for everybody...